skyve-war/src/main/java/modules/admin/UserLoginRecord/UserLoginRecordBizlet.java

```diff
@@ -0,0 +1,129 @@
+package modules.admin.UserLoginRecord;
+
+import java.util.Locale;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.skyve.CORE;
+import org.skyve.EXT;
+import org.skyve.domain.app.AppConstants;
+import org.skyve.impl.bind.BindUtil;
+import org.skyve.impl.cdi.GeoIPService;
+import org.skyve.impl.util.UtilImpl;
+import org.skyve.metadata.SortDirection;
+import org.skyve.metadata.customer.Customer;
+import org.skyve.metadata.model.document.Bizlet;
+import org.skyve.metadata.module.JobMetaData;
+import org.skyve.metadata.module.Module;
+import org.skyve.persistence.DocumentQuery;
+import org.skyve.persistence.Persistence;
+import org.skyve.util.SecurityUtil;
+
+import jakarta.inject.Inject;
+import modules.admin.User.UserExtension;
+import modules.admin.UserLoginRecord.jobs.DifferentCountryLoginNotificationJob;
+import modules.admin.domain.User;
+import modules.admin.domain.UserLoginRecord;
+
+public class UserLoginRecordBizlet extends Bizlet<UserLoginRecordExtension> {
+
+	@Inject
+	private transient GeoIPService geoIPService;
+
+	private static final String COUNTRY = "country";
+	private static final String IP_ADDRESS = "ipAddress";
```

Comment:
Please remove these statics - you don't need them as you have access to `UserLoginRecord` getters in the bizlet. But even if you didn't, you could use the `UserLoginRecord` static attribute names.